chatbot.js

```diff
@@ -46,6 +47,8 @@ const getBotReply = (username, message, parsedMessage) => {
     places.getPlaces(api.postBotReply, username, cityName);
   } else if (startsWithString === constants.BOT_ACTIONS.DEFINE) {
     define.define(api.postBotReply, username, cityName);
+  } else if (startsWithString === constants.BOT_ACTIONS.INSULT) {
+    insult.insult(api.postBotReply, cityName);
```

Comment:
@yashmehrotra `username` is the name of the person who gave the command and `cityName` is the 3rd word, so if I say `@osdc-bot insult @526avijitgupta`, `username => @ryzokuken` and `cityName => @526avijitgupta`. I thought it would be counterproductive if one ended up insulting themselves :b